src/popup/app.scss

```
@@ -0,0 +1,15 @@
+.update-banner {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	gap: 4px;
+	margin: 10px 10px 0;
+	padding: 8px 10px;
+
+	border: 2px solid var(--primary);
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)